llvm/test/Analysis/CostModel/RISCV/fp-min-max-abs.ll

```diff
@@ -1,28 +1,28 @@
 ; NOTE: Assertions have been autogenerated by utils/update_analyze_test_checks.py
 ; RUN: opt < %s -passes="print<cost-model>" 2>&1 -disable-output -S -mtriple=riscv64 -mattr=+v,+f,+d,+zfh,+zvfh | FileCheck %s --check-prefixes=CHECK,ZVFH
```

Comment:
Can you add zvfbfmin to the ZVFH run line too? I think that matches the other tests with bfloat

Reply:
done